tests/drivers/build_all/display/app.overlay

```diff
@@ -209,6 +209,25 @@
 				vcomh-voltage = <0>;
 				mipi-mode = "MIPI_DBI_MODE_SPI_4WIRE";
 			};
+
+			test_mipi_dbi_ssd1351: ssd1351@13 {
+				compatible = "solomon,ssd1351";
+				mipi-max-frequency = <10000000>;
+				reg = <13>;
+				width = <128>;
+				height = <128>;
+				oscillator-freq = <0x0>;
+				display-offset = <0x0>;
+				multiplex-ratio = <0x0>;
+				phase-length = <0x0>;
+				vcomh-voltage = <0x0>;
+				current-att = <0x0>;
+				precharge-time = <0x0>;
```

Comment:
0 should not be allowed?

Reply:
most values everywhere for the build tests are invalid on every drivers